infra/ballista/build/lambda-handler.py

```diff
@@ -0,0 +1,80 @@
+import os
+import subprocess
+import logging
+import sys
+import shutil
+import time
+
+
+logging.getLogger().setLevel(logging.DEBUG)
+__stdout__ = sys.stdout
+
+IS_COLD_START = True
+
+def handler(event, context):
+    """An AWS Lambda handler that runs the provided command with bash and returns the standard output"""
+    shutil.rmtree("/tmp", ignore_errors=True)
+    start = time.time()
+    global IS_COLD_START
+    is_cold_start = IS_COLD_START
+    IS_COLD_START = False
+    # start scheduler
+    process_s = subprocess.Popen(["/opt/ballista/ballista-scheduler"],
+                                stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    logging.debug("scheduler starts")
+    # wait till the scheduler is up and running
+    return_code=1
+    while return_code:
+        res = subprocess.run(["nc", "-z", "localhost", "50050"], capture_output=True)
+        return_code = res.returncode
+        logging.debug(res)
+        
+    process_e = subprocess.Popen(["/opt/ballista/ballista-executor", "-c" ,"4"],
+                                 stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    logging.debug("executor.starts")
+    return_code=1
+    while return_code:
+        res = subprocess.run(["nc", "-z", "localhost", "50051"], capture_output=True)
+        return_code = res.returncode
+        logging.debug(res)
```

Comment:
could we maybe factor this out into an `init` function ?

Reply:
yes I can. actually it was like that at the beginning and then evolve towards the current version